docs/job-scheduling.md

```diff
@@ -256,6 +256,7 @@ and either putting a file named `fairscheduler.xml` on the classpath, or setting
 
 {% highlight scala %}
 conf.set("spark.scheduler.allocation.file", "/path/to/file")
+conf.set("spark.scheduler.allocation.file", "hdfs:///path/to/file")
```

Comment:
I think HDFS also supports reading the local file. Could you mention it in the above document? e.g., "The file path can either be a local file path or HDFS file path."